nntrainer/src/layer.cpp

```diff
@@ -198,6 +198,15 @@ void Layer::setProperty(const PropertyType type, const std::string &value) {
       bias_initializer = (WeightInitializer)parseType(value, TOKEN_WEIGHT_INIT);
     }
     break;
+  case PropertyType::input_layers:
+    if (!value.empty()) {
+      std::regex reg("\\,+");
```

Comment:
make "\\,+" as a const string?

Reply:
Want to use ',' as a delimiter and others like '+','-','*','%' will be added.

Reply:
Yes, thats why declare it as a string as a separate variable.
Also, might consider moving all of this to parse_util later.